server/models/role.js

```diff
@@ -0,0 +1,41 @@
+
+/**
+ * The roles of a Bits of Good member.
+ * @enum {string}
+ */
+const Role = {
+  EXEC: "exec", // Exec members
+  LEADER: "leader", // Leadership: PM's, EM's, etc.
+  MEMBER: "member" // An ordinary member
+};
+
+/**
+   * Returns whether a string is a valid role.
+   * @param {string} string A string
+   */
+Role.isRole = function isRole(string) {
+  return Object.values(Role).indexOf(string) !== -1;
+};
+
+/**
+   * Returns whether a member with the given role is at the given permission level.
+   * @param {Role} role The role of the member
+   * @param {Role} permissionLevel The permission level
+   */
+Role.hasPermission = function hasPermission(role, permissionLevel) {
```

Comment:
Is it necessary to have a permission level in addition to role? I would think your permission level is contained within the role.

Reply:
Also, I think this method should be part of the `Member` model. Roles are tied to Members so we can abstract this away.

Reply:
Good point. It'll be even more confusing for people who contribute later on, so I just ended up removing it.